notebooks/notebooks/paper/katago-new-ckpt-evals.ipynb

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.            cdf = df.query("victim_net_type == @net_type and victim_visits == @vv")
</pre><p>i've definitely asked this before but what does <code>cdf</code> stand for again? is it "current df"? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Haha yeah cdf stands for current df. Changed to `sub_df`.